service/history/queues/queue_scheduled.go

```diff
@@ -39,7 +39,7 @@ import (
 	"go.temporal.io/server/common/persistence"
 	"go.temporal.io/server/common/quotas"
 	"go.temporal.io/server/common/timer"
-	"go.temporal.io/server/service/history/shard"
+	hshard "go.temporal.io/server/service/history/shard"
```

Comment:
We only have the concept in history so I guess it is ok just use shard?

Reply:
The problem is that the `shard` identifier conflicts with many variables named `shard`. So I have two options:
* rename the identifier to something else
* rename all variables to something else (for example, `shardContext`)

I think renaming the identifier is more readable.

What do you think?

Reply:
👍 I guess I am ok with either option. cc @yux0

Reply:
I see after the linter plugin.